curriculum/challenges/english/25-front-end-development/workshop-plant-nursery-catalog/67349ba230cff237c0f2bdf2.md

```diff
@@ -0,0 +1,84 @@
+---
+id: 67349ba230cff237c0f2bdf2
+title: Step 4
+challengeType: 0
+dashedName: step-4
+---
+
+# --description--
+
+Map objects remember the order in which the keys are inserted in the map. To set a key-value pair in your map object, you can use the `set` method. The following sets the key `"name"` to the value `"Jerry"` in the `myMap` map object.
+
+```js
+myMap.set("name", "Jerry");
+```
+
+Set the `"ballerina"` key to the value of your `ballerina` object in your `catalog` map. Then, log your catalog to the console.
+
+Note that to see your map object properly you'll have to open the developer console (you can do it by pressing the F12 key on your keyboard).
```

Comment:
```suggestion
Note that to see your map object properly you'll have to open the developer console.
```

I wouldn't put that, different browsers have different methods - hopefully, they know at this point.